blocks/library/freeform/freeform-block.js

```diff
@@ -96,6 +99,18 @@ export default class FreeformBlock extends wp.element.Component {
 		this.state = {
 			empty: ! props.value || ! props.value.length,
 			activeButtons: { },
+			activeFormat: null,
+			formats: [
+				{ text: 'Paragraph', value: 'p', textStyle: () => '' },
+				{ text: 'Heading 1', value: 'h1', textStyle: () => '' },
+				{ text: 'Heading 2', value: 'h2', textStyle: () => '' },
+				{ text: 'Heading 3', value: 'h3', textStyle: () => '' },
+				{ text: 'Heading 4', value: 'h4', textStyle: () => '' },
+				{ text: 'Heading 5', value: 'h5', textStyle: () => '' },
+				{ text: 'Heading 6', value: 'h6', textStyle: () => '' },
+				{ text: 'Preformatted', value: 'pre', textStyle: () => '' },
+			],
+			handleFormatChange: null,
```

Comment:
They are drawn from the editor but I suppose I could keep them outside of the state. I do need to re-render when I set the formats array (in onInit) but I suppose I could force that instead of updating state.